Sources/MarathonCore/Create.swift

```diff
@@ -38,12 +38,12 @@ extension CreateError: PrintableError {
 
 internal final class CreateTask: Task, Executable {
     private typealias Error = CreateError
-
+    
```

Comment:
Please unstage these changes 😉 Pro tip is to make Xcode automatically remove this kind of whitespace for you:
![screen shot 2017-06-04 at 18 38 59](https://cloud.githubusercontent.com/assets/2466701/26763555/2f685b3a-4955-11e7-9f76-462eec88c77e.png)